texar/torch/modules/encoders/xlnet_encoder.py

```diff
@@ -269,6 +269,9 @@ def param_groups(self,
 
     @property
     def output_size(self):
+        r"""The feature size of :meth:`_forward` result
```

Comment:
Let's not mention `_forward` as we don't expect users to use that method. And let's be consistent with the docstring, in most of then you say "output", but here you say "result".